projects/1.20-fabric/assets/more-wizards/morewizardsmod/lang/zh_cn.json

```
@@ -0,0 +1,41 @@
+{
+  "item.morewizardsmod.more_wizards_lightning_spell_book": "雷电魔法书",
+  "item.morewizardsmod.more_wizards_soul_spell_book": "灵魂魔法书",
+  "item.morewizardsmod.lightning_staff": "雷电权杖",
+  "item.morewizardsmod.netherite_lightning_staff": "下界合金雷电权杖",
+  "item.morewizardsmod.soul_staff": "灵魂权杖",
+  "item.morewizardsmod.netherite_soul_staff": "下界合金灵魂权杖",
+
+  "item.morewizardsmod.lightning_wizard_armor_t1_head": "雷电学徒帽",
+  "item.morewizardsmod.lightning_wizard_armor_t1_chest": "雷电学徒长袍 ",
+  "item.morewizardsmod.lightning_wizard_armor_t1_legs": "雷电学徒护腿",
+  "item.morewizardsmod.lightning_wizard_armor_t1_feet": "雷电学徒靴子",
+  "item.morewizardsmod.lightning_wizard_armor_t2_head": "雷电魔法师帽",
+  "item.morewizardsmod.lightning_wizard_armor_t2_chest": "雷电魔法师长袍",
+  "item.morewizardsmod.lightning_wizard_armor_t2_legs": "雷电魔法师护腿",
+  "item.morewizardsmod.lightning_wizard_armor_t2_feet": "雷电魔法师靴子",
+
+  "item.morewizardsmod.soul_wizard_armor_t1_head": "灵魂学徒帽",
+  "item.morewizardsmod.soul_wizard_armor_t1_chest": "灵魂学徒长袍",
+  "item.morewizardsmod.soul_wizard_armor_t1_legs": "灵魂学徒护腿",
+  "item.morewizardsmod.soul_wizard_armor_t1_feet": "灵魂学徒靴子",
+  "item.morewizardsmod.soul_wizard_armor_t2_head": "灵魂魔法师帽",
+  "item.morewizardsmod.soul_wizard_armor_t2_chest": "灵魂魔法师长袍",
+  "item.morewizardsmod.soul_wizard_armor_t2_legs": "灵魂魔法师护腿",
+  "item.morewizardsmod.soul_wizard_armor_t2_feet": "灵魂魔法师靴子",
+
+  "spell.morewizardsmod.lightning_strike.name": "雷霆一击",
+  "spell.morewizardsmod.lightning_strike.description": "召唤闪电束劈击一个目标3次",
+  "spell.morewizardsmod.discharge.name": "雷电释放",
+  "spell.morewizardsmod.discharge.description": "释放一股强大的能量波，对施法者周围的所有敌人造成3次伤害",
+  "spell.morewizardsmod.thunder_summoning.name": "天雷罚",
+  "spell.morewizardsmod.thunder_summoning.description": "从天空中召唤一道强大的落雷，在小范围内劈击所有目标5次",
+  "spell.morewizardsmod.ancient_soul_summoning.name": "灵魂飞弹",
+  "spell.morewizardsmod.ancient_soul_summoning.description": "远程攻击，造成单体伤害",
+  "spell.morewizardsmod.soul_release.name": "魂焰吐息",
+  "spell.morewizardsmod.soul_release.description": "持续喷射灵魂火焰，对一定射程内的目标持续造成伤害",
+  "spell.morewizardsmod.soul_beam.name": "灵魂光束",
+  "spell.morewizardsmod.soul_beam.description": "持续喷射灵魂激光，对路径内所有目标持续造成伤害",

```

原文`Channel soul energy to blast your enemies with a powerful beam`

```suggestion
  "spell.morewizardsmod.soul_beam.description": "引导一束灵魂激光，对路径内所有目标持续造成伤害",
```
Channel没翻译出来